### PR TITLE
:construction_worker: Use reusable workflows

### PR DESCRIPTION
**Pull request by 0x46616c6b**

### CODE REVIEW

File: .github/workflows/workflow_integration.yml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+          MOLECULE_DISTRO: ${{ matrix.distro }}
+
+  release:
+    name: Ansible Galaxy
+    runs-on: ubuntu-20.04
+    needs: [test]
+    if: github.ref_name == inputs.default-branch
+    steps:
+      - name: Publish to Ansible Galaxy
+        uses: robertdebock/galaxy-action@1.1.1
```
Comment (t2d) [inline .github/workflows/workflow_integration.yml:107]:
```suggestion
        uses: robertdebock/galaxy-action@1.2.0
```

File: .github/workflows/workflow_integration.yml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        env:
+          PY_COLORS: '1'
+          ANSIBLE_FORCE_COLOR: '1'
+          MOLECULE_DISTRO: ${{ matrix.distro }}
+
+  release:
+    name: Ansible Galaxy
+    runs-on: ubuntu-20.04
+    needs: [test]
+    if: github.ref_name == inputs.default-branch
```
Comment (t2d) [inline .github/workflows/workflow_integration.yml:104]:
```suggestion
    if: github.ref_name == inputs.default-branch || contains(github.ref, 'refs/tags/')
```

File: tasks/main.yml [hunk 3]
```diff
@@ -1,34 +1,35 @@
 ---
 
 - name: Get checksums from github.com
-  set_fact:
+  ansible.builtin.set_fact:
```
Comment (t2d) [inline tasks/main.yml:4]: I would prefer to use a `.ansible-lint` instead like https://github.com/systemli/ansible-drift/blob/main/.ansible-lint

Comment (0x46616c6b) [reply tasks/main.yml]: Try to fix this independently from this PR. See #26

File: .github/workflows/workflow_integration.yml [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+  release:
+    name: Ansible Galaxy
+    runs-on: ubuntu-20.04
+    needs: [test]
+    if: github.ref_name == inputs.default-branch || contains(github.ref, 'refs/tags/')
+    steps:
+      - name: Publish to Ansible Galaxy
+        uses: robertdebock/galaxy-action@1.2.0
+        with:
+          galaxy_api_key: ${{ secrets.galaxy-token }}
```
Comment (t2d) [inline .github/workflows/workflow_integration.yml:109]:
```suggestion
          galaxy_api_key: ${{ secrets.galaxy-token }}
          git_branch: main
```

**Pull request merged**